### PR TITLE
v0.6.0 - screen, punchbag, RAM saving

### PR DESCRIPTION
**Pull request by dmcomm**

### CODE REVIEW

**Review by mechawrench — COMMENTED** [review]
Tested this out with pronged and RTB(PenX) with success.

Everything looks great, just a few comments on documentation items

**Comment by mechawrench** [comment]
Let me know if you think the comments above need addressed, if not I can mark as approved straight away

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 [![Pylint](https://github.com/mechawrench/wificom-lib/actions/workflows/pylint.yml/badge.svg)](https://github.com/mechawrench/wificom-lib/actions/workflows/pylint.yml)
 
-This library enables an RP2040 device using dmcomm-python (CircuitPython 8.x/7.3.x) to connect to https://wificom.dev to enable DigiRom reading and writing via HTTP API calls and MQTT for device interactions.
+This library enables an RP2040 device using dmcomm-python (CircuitPython 8.x) to connect to https://wificom.dev to enable DigiRom reading and writing via HTTP API calls and MQTT for device interactions.
```
[the remark this answers is not in the document]
Comment (dmcomm) [reply README.md:11]: Probably just beta 3 and beta 4 at the moment, but I was putting exact versions tested in the release notes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.6.0]
+### Added
```
[the remark this answers is not in the document]
Comment (dmcomm) [reply CHANGELOG.md:8]: Added a note about that

**Review by mechawrench — APPROVED** [review]

**Pull request merged**